daml-script/daml/BUILD.bazel

```
@@ -3,29 +3,30 @@
 
 # TODO Once daml_compile uses build instead of package we should use
 # daml_compile instead of a genrule.
+load("@build_environment//:configuration.bzl", "ghc_version", "sdk_version")
 
 genrule(
     name = "daml-script",
-    srcs = glob(["**/*.daml"]) + ["//:VERSION"],
+    srcs = glob(["**/*.daml"]),
     outs = ["daml-script.dar"],
     cmd = """
       set -eou pipefail
       TMP_DIR=$$(mktemp -d)
       mkdir -p $$TMP_DIR/daml/Daml
       cp -L $(location Daml/Script.daml) $$TMP_DIR/daml/Daml
       cat << EOF > $$TMP_DIR/daml.yaml
-sdk-version: $$(cat $(location //:VERSION))
+sdk-version: {}
 name: daml-script
 source: daml
-version: $$(cat $(location //:VERSION))
+version: {}
```

Two unnamed placeholders in the same interpolation scare me even more.
```suggestion
version: {ghc_version}
```